content/docs/5.1/concepts/tikv-in-5-minutes.md

```diff
@@ -0,0 +1,219 @@
+---
+title: TiKV in 5 Minutes
+description: Get started using TiKV in 5 Minutes
+menu:
+    "5.1":
+        parent: Get Started
+        weight: 1
+---
+
```

Comment:
Provide a short description for this document, and the goal.

Reply:
fixed